docs/build/graphql.md

```diff
@@ -268,6 +268,18 @@ type User @entity {
 }
 ```
 
+### JSON field indexes
+
+By default JSON fields will have a database index to improve querying performance. This can be disabled by specifying the `indexed` argument on the directive.
+This is useful if you are using Cockroach DB as there can be some perfomance issues with inserting JSON data with an index
```

Comment:
@stwiname  are there any issues in general using indexes on Cockroach?

Reply:
Just the gin index and Jsonb data